### ISSUE TITLE
Installation warning

### ISSUE THREAD

**Issue by mhulse** [LinusU/fs-xattr#20]
```bash
$ npm install fs-xattr -S
  CXX(target) Release/obj.target/xattr/src/async.o
  CXX(target) Release/obj.target/xattr/src/error.o
  CXX(target) Release/obj.target/xattr/src/sync.o
../src/sync.cc:46:59: warning: 'NewInstance' is deprecated [-Wdeprecated-declarations]
  v8::Local<v8::Object> actualBuffer = bufferConstructor->NewInstance(3, constructorArgs);
                                                          ^
/Users/mhulse/.node-gyp/7.3.0/include/node/v8.h:3288:31: note: 'NewInstance' has been explicitly marked deprecated here
                Local<Object> NewInstance(int argc, Local<Value> argv[]) const);
                              ^
1 warning generated.
  CXX(target) Release/obj.target/xattr/src/util.o
  CXX(target) Release/obj.target/xattr/src/xattr.o
  SOLINK_MODULE(target) Release/xattr.node
merch-name-cli@0.1.0 /Users/mhulse/dev/bitbucket/pixelfoot/merch-name-cli
└── fs-xattr@0.1.14
```

Not sure if it's just me though. Could be my setup. Just wanted to FYI.

Node v7.3.0, MacOS

**Comment by LinusU** [comment]
Ah, this was fixed but never released, thank you for notifying me!

**Comment by LinusU** [comment]
Published as 0.1.15

**Comment by mhulse** [comment]
Thanks! I really appreciate it.  :)
